CLAUDE.md

```
@@ -0,0 +1,24 @@
+
+## Active Technologies
+- TypeScript 5.x with React 18.x and Next.js 14.x + Next.js, React, Tailwind CSS, shadcn/ui components, HTML5 Canvas API (001-i-want-to)
+
+## Project Structure
+```
+src/
+tests/
+```
+
+## Commands
+npm test [ONLY COMMANDS FOR ACTIVE TECHNOLOGIES][ONLY COMMANDS FOR ACTIVE TECHNOLOGIES] npm run lint
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "only commands for active technologies"

(MD052, reference-links-images)